src/main/java/com/cloudogu/smeagol/wiki/domain/DisplayName.java

```diff
@@ -0,0 +1,57 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+
+import java.util.Objects;
+
+/**
+ * Simple string value object.
+ */
+public class DisplayName {
+
+    private final String value;
+
+    private DisplayName(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the display name.
+     *
+     * @return string representation
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Creates a display name from a string.
+     *
+     * @param value string representation of display name
+     *
+     * @return repository name
+     */
+    public static DisplayName valueOf(String value) {
+        Preconditions.checkArgument(!Strings.isNullOrEmpty(value), "display name can not be null or empty");
+        return new DisplayName(value);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)